packages/react-dom/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "homepage": "https://reactjs.org/",
   "dependencies": {
+    "dom-accessibility-api": "^0.4.3",
```

Comment:
We've been trying to avoid adding more runtime deps. Since that makes the future move to ESM harder to coordinate. See also my other comment.